README.md

```diff
@@ -103,26 +103,40 @@ you'll get an error. You can fix this by either logging in (via `docker login`)
 
 Building
 -
-To build the manual you will need go-md2man.
+To build the `skopeo` binary you need at least Go 1.5 because it uses the latest `GO15VENDOREXPERIMENT` flag.
+
+### Building in a container
+If your system has a `docker` command and can run Linux containers,
 ```sh
-$ sudo apt-get install go-md2man
+$ make binary # Or (make all) to also build documentation, see below.
```

Comment:
The docs make more sense now, but when I actually do run `make binary` I get:

    ....
    Step 4 : WORKDIR /src/github.com/projectatomic/skopeo
     ---> Running in bf527540083c
     ---> f817788091bf
    Removing intermediate container bf527540083c
    Successfully built f817788091bf
    docker run --rm --security-opt label:disable -v $(pwd):/src/github.com/projectatomic/skopeo \
    		skopeobuildimage make binary-local  BUILDTAGS='btrfs_noversion libdm_no_deferred_remove     containers_image_openpgp'
    CGO_ENABLED=1 GOARCH=amd64 GOOS=darwin go build -ldflags "-X     main.gitCommit=07c798ff82f82a68571b03eb9be3469baf483d93" -gcflags "" -tags "btrfs_noversion     libdm_no_deferred_remove containers_image_openpgp" -o skopeo ./cmd/skopeo
    # net
    could not determine kind of name for C.AI_MASK
    Makefile:60: recipe for target 'binary-local' failed
    make: *** [binary-local] Error 2
    make: *** [binary] Error 2